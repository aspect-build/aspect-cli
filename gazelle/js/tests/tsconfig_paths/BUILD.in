# gazelle:generation_mode update_only
# gazelle:js_tsconfig enabled
