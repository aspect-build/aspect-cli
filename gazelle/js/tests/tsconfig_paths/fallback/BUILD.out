load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

ts_project(
    name = "fallback",
    srcs = [
        "f1.ts",
        "f2/a.ts",
    ],
    deps = ["//lib/a"],
)
